packages/database/src/migrations/20201116024506-RemoveMapOverlayUNFPA-modifies-data.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import { insertObject } from '../utilities/migration';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return db.runSql(`
+    DELETE FROM map_overlay_group_relation where child_id IN (SELECT id FROM map_overlay_group where name = 'Services provided');
```

Comment:
Hey @PShoe ! This will remove `Services Provided` from all projects. What we want to do instead is remove it from the `unfpa` project specifically.

Map overlays are attached to projects via the `mapOverlay.projectCodes` column. Thus, we should remove all overlays belonging to the `Services Provided` group from the `unfpa` project.  After this is done, the group will be empty for UNFPA and it shouldn't show up in the front end.

Please let me know if you need more help! 🙌

Reply:
@kael89 was this more of what you had in mind? 
Thank you for the extra-effort review and sql help 🙏